crates/typst-layout/src/math/mod.rs

```diff
@@ -110,9 +116,17 @@ pub fn layout_equation_block(
 
     let mut locator = locator.split();
     let mut ctx = MathContext::new(engine, &mut locator, styles, regions.base(), &font);
+
+    let local = [
+        EquationElem::set_script_scale(ctx.constants.script_percent_scale_down()),
+        EquationElem::set_sscript_scale(ctx.constants.script_script_percent_scale_down()),
+    ]
+    .map(|p| p.wrap());
```

Comment:
could we deduplicate that with the one in layout_equation_inline?

Reply:
Will do